source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -471,9 +471,10 @@ ContextImpl::ContextImpl(Stats::Scope& scope, const Envoy::Ssl::ContextConfig& c
   }
 
   // Add hardcoded cipher suites from the TLS 1.3 spec:
-  // https://tools.ietf.org/html/rfc8446
-  stat_name_set_->rememberBuiltins(
-      {"TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384", "TLS_AES_128_GCM_SHA256"});
+  // https://tools.ietf.org/html/rfc8446#appendix-B.4
+  stat_name_set_->rememberBuiltins({"TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384",
+                                    "TLS_AES_128_CCM_SHA256", "TLS_AES_256_CCM_8_SHA256",
```

Comment:
`TLS_AES_128_CCM_SHA256` and `TLS_AES_256_CCM_8_SHA256` are not supported by BoringSSL, so we can drop them, IMHO.

Reply:
That makes sense; if you remove those, can you just leave a comment indicating they are in-spec but not in BoringSSL.

Reply:
Yep, done!